src/Eccube/Resource/template/admin/Store/plugin_owners_search.twig

```diff
@@ -34,6 +34,7 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
                 {% elseif item.update_status == 4 %}
                     <a class="btn btn-success btn-xs" href="{{ item.product_url }}" target="_blank">購入</a>
                 {% else %}
+                    <a class="btn btn-xs searchPluginModal" href="{{ url('admin_store_plugin_install_confirm', {'id': item.product_id}) }}" data-id="{{ item.product_id }}">詳細表示</a>
```

Comment:
need href url ??

Reply:
yes, because  URL : /admin/store/plugin/{id}/confirm, so have to generate URL by php, using JS to generate so complex